README.md

```diff
@@ -1 +1,20 @@
-# matrix_convert_to_zero
\ No newline at end of file
+# Matrix: Convert to zero
+Changes the corresponding row and column to have '0' values if the value is '0' in the input matrix.
+
+## Exercise
+* Design and implement a method that updates the input matrix. The input matrix i.e. two-dimensional array contains only '0's and '1's. If any number in the array is found to be '0', the method updates all the numbers in the corresponding row as well as the corresponding column to be '0'.
+* For example if the input is:
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 1 1
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 0 1
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 1 1
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 1 0
+should get updated to:
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 0 0
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0 0 0
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 0 0
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0 0 0
+
+* Share and explain the time and space complexities for your solution.
+* If you think of multiple approaches to solve the problem, implement the solution which minimizes space complexity. Explain the other approaches, and your decision in comments above the code.
+
+**Note**: Do not use Ruby provided functionality.
```

Comment:
What functionality are we talking about here?

I could definitely see this algorithm being something that they write using standard Ruby constructs such as `each` or other enumerable methods, and I'm not sure that would detract from their understanding.

I don't know if there are any Ruby libraries for dealing with matrices (I assume there are some Gems, not sure about in the standard lib), but that would be the most obvious thing to restrict them from using.

Reply:
Fair point! Removed.